### PR TITLE
Adds Query for Sheet 101 + WS_INFO Cleanup

### PR DESCRIPTION
**Pull request by ascii-dev**
## Description
Generates the new 2020 query - sheet 101 and also cleans up the new queries information in `WS_INFO`.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshot
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ services:
     depends_on:
       - db
     environment:
-      - DATABASE_URL=postgresql://gmmp:gmmp@db:5432/gmmp
+      - DATABASE_URL=postgresql://asciicfa:@host.docker.internal:5432/gmmp
       - DJANGO_SECRET_KEY=somethingsecret
```
Comment (esirK) [inline docker-compose.yml]: I think you can add 
```
- DJANGO_DEBUG=${DJANGO_DEBUG:-False} # For testing deploys
- GMMP_REPORTS_HISTORICAL_YEAR=2015
```
to be able to generate all sheets

File: reports/report_builder.py [hunk 2]
```diff
@@ -3029,7 +3029,24 @@ def ws_100(self, ws):
...
     def ws_101(self, ws):
-        return
+        counts_list = []
+        for _, model in sheet_models.items():
+            counts = Counter()
+            sex = model.journalist_field_name() + '__sex'
+
+            rows = model.objects \
+                .values('covid19', 'topic', sex, 'country') \
+                .filter(**{model.journalist_field_name() + '__sex__in': self.male_female_ids}) \
```
Comment (kilemensi) [inline reports/report_builder.py:3057]: What effect does this have i.e. is it possible to have sex not defined in `male_female_ids`?

File: reports/report_details.py [hunk 3]
```diff
@@ -1527,7 +1527,8 @@ def add_transnational_to_regions(regions):
             'name': '98',
             'title': 'Twitter - Images',
             'desc': '',
-            'reports': [],
+            'historical': '98',
+            'reports': ['global'],
```
Comment (kilemensi) [inline reports/report_details.py]: Why are we changing this? It's not related to this query.

File: reports/report_details.py [hunk 4]
```diff
@@ -1541,7 +1542,7 @@ def add_transnational_to_regions(regions):
             'name': '100',
             'title': 'Is this story related to Covid19. By major topic, by medium',
             'desc': 'Is this story related to Covid19. By major topic, by medium',
-            'reports': [],
+            'reports': ['global'],
```
Comment (kilemensi) [inline reports/report_details.py]: Why are we changing this? It's not related to this query.

File: reports/report_details.py [hunk 5]
```diff
@@ -1555,7 +1556,7 @@ def add_transnational_to_regions(regions):
             'name': '101',
             'title': 'Covid stories:  reporters, by sex',
             'desc': 'Covid stories:  reporters, by sex',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'desc': '',
```
No description in queries sheet.

**Review by kilemensi — APPROVED** [review]
🚀 @ascii-dev 


-------

Resolve those merge conflicts though...

File: reports/report_details.py (hunk 3 above)
Comment (ascii-dev) [reply reports/report_details.py]: That was the first query I tried working on when I created this PR, that's why I made this change. I'll update to revert the change. @kilemensi

File: reports/report_details.py (hunk 4 above)
Comment (ascii-dev) [reply reports/report_details.py]: That was also a query I tried working on when I created this PR, that's why I made this change. @kilemensi

File: reports/report_builder.py (hunk 2 above)
Comment (ascii-dev) [reply reports/report_builder.py:3057]: From what I understand at other places where this was used, I think there are stories that don't have `sex` defined. I'm not sure though but that was what I inferred from context. @kilemensi

**Pull request merged**